Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -584,13 +585,13 @@ def _update_stats_model(self):
                 stats = slot.stats
 
             if stats is not None:
-                for stat in stats:
+                for stat, scorer in zip(stats, self.scorers):
                     item = QStandardItem()
                     if stat.success:
                         item.setText("{:.3f}".format(stat.value[0]))
                     else:
                         item.setToolTip(str(stat.exception))
-                        has_missing_scores = True
+                        has_missing_scores = scorer.name in self.shown_scores
```

Comment:
Should this be `has_missing_scores = has_missing_scores or scorer.name in self.shown_scores`? It can already be `True`.

Reply:
Good catch.